__tests__/articles.test.js

```diff
@@ -2,6 +2,7 @@ const request = require("supertest");
 const db = require("../db/connection");
 const seed = require("../db/seeds/seed");
 const app = require("../app");
+const jestSorted = require("jest-sorted");
```

Comment:
this doesn't need to be imported, the config in package.json should be updated

Reply:
<img width="250" alt="Screenshot 2024-08-26 at 13 29 17" src="https://github.com/user-attachments/assets/0bbfc2e1-0b7e-4840-944a-d7fc04fbda94">